lib/book.rb

```diff
@@ -0,0 +1,25 @@
+class Book
+  attr_reader :title, :author, :publication_year
+  attr_accessor :popularity
+
+  def initialize(info)
+    @title = info[:title]
+    @author = "#{info[:author_first_name]} #{info[:author_last_name]}"
+    @publication_year = info[:publication_date].split.last.to_i
+    @popularity = 0
+    @checked_out = false
```

Comment:
Love this design decision to have a book keep track of its `checked_out` status 👍